share/zm-rpcapi.sysv

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+#
+### BEGIN INIT INFO
+# Provides:          zm-rpcapi
+# Required-Start:    $network $local_fs
+# Required-Stop:     $network $local_fs
+# Should-Start:      mysql postgresql
+# Should-Stop:       mysql postgresql
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: A JSON-RPC frontend for Zonemaster Backend
+# Description:       zm-rpcapi lets you add new tests and check for results in
+#                    the the Zonemaster Backend database
+### END INIT INFO
+
+BINDIR=${ZM_BACKEND_BINDIR:-/usr/local/bin}
+LOGFILE=${ZM_BACKEND_LOGDIR:-/var/log/zonemaster}/zm-starman-error.log
+PIDFILE=${ZM_BACKEND_PIDDIR:-/var/run/zonemaster}/zm-starman.pid
+LISTENIP=${ZM_BACKEND_LISTENIP:-127.0.0.1}
+USER=${ZM_BACKEND_USER:-zonemaster}
+GROUP=${ZM_BACKEND_GROUP:-zonemaster}
+
+STARMAN=`PATH="$PATH:/usr/local/bin" /usr/bin/which starman`
+
+start () {
+    $STARMAN --user=$USER --group=$GROUP --error-log=$LOGFILE --pid=$PIDFILE --listen=$LISTENIP:5000 --preload-app --daemonize $BINDIR/zonemaster_backend_rpcapi.psgi
```

Comment:
Having port 5000 as default is fine, but I think it would make sense to create:

LISTENPORT=${ZM_BACKEND_LISTEN_PORT:-5000}

and to refer to that as well.

Reply:
I wanted to limit the scope of the change. But sure, I'll add that.

Reply:
By the way, the environment variable overrides are intended to only be used when developing Zonemaster-Backend. Though if someone wants to use them in some other context, I won't mind.